core/cap-0035.md

```diff
@@ -624,6 +627,12 @@ A new operation, `SetTrustLineFlagsOp`, is proposed in this cap to make
 trustline flag modification simpler. Additionally the `AllowTrustOp` operation is
 semantically intended to change authorization and clawback is outside it's scope.
 
+### Set TrustLine Flags Operation
+The `SetTrustLineFlagsOp` will make it easier for issuers to modify trustline flags.
+In the context of clawback, an issuer can clear the `CLAWBACK_ENABLED_FLAG` from a trustline
+without knowledge of the current state of the other flags. This is not possible with
+`AllowTrustOp` because it sets the trustline flags to the value passed in by `AllowTrustOp`.

```

Comment:
💡 I think we could elaborate a little to why this can't be solved using the existing operation. It overlaps a little with another section in the document but it helps provide a fuller picture.
```suggestion
`AllowTrustOp` because it sets the trustline flags to the value passed in by `AllowTrustOp`.
It is also not possible to add the existing flag to the `AllowTrustOp` operation without
breaking existing users of that operation since they will not be expecting to include the
current state of the `CLAWBACK_ENABLED_FLAG` in the flags field.
```